src/generators/android/templates/RudderTyperUtils.java.hbs

```diff
@@ -39,10 +39,10 @@ public final class RudderTyperUtils {
             return props;
         }
 
-        List p = new ArrayList<>();
+        List<Object> p = new ArrayList<>();
         for(Object item : props) {
             if (item instanceof List) {
-                p.add(serializeList((List) item));
+                p.add(serializeList((List<?>) item));
```

Comment:
This one is not flagged by `-Werror`, but highlighted by IDE as a raw usage. Updated to match the input type of `serializeList`.